index.js

```diff
@@ -57,6 +57,17 @@ FileBin.prototype.write = function (fileName, data) {
   });
 };
 
+FileBin.prototype.rename = function (oldFileName, newFileName) {
+  var oldFullPath = path.join(this.base, oldFileName);
+  var newFullPath = path.join(this.base, newFileName);
+  return new RSVP.Promise((resolve, reject) => {
+    fs.rename(oldFullPath, newFullPath, (error) => {
```

Comment:
If you look in #2, the example returns the file object as the first argument, then the new path, and then the old path. At the very least, it's safe to assume that the user would want the new name as the earlier argument and just omit the old name if necessary.

In order to get the full file, you'd have to `this.find(newFileName)` if rename was successful and then resolve the promise based on the result.